website/docs/reference/dbt-jinja-functions/cross-database-macros.md

```diff
@@ -884,6 +887,29 @@ For databases that support it, this macro will return `NULL` when the cast fails
 
 ## Date and time functions
 
+### date
+
+**Availability**:
+dbt v1.8 or later. For more information, select the version from the documentation navigation menu.
+
+<VersionBlock firstVersion="1.8">
+
+__Args__:
+
+ * `year`: an integer
+ * `month`: an integer
+ * `day`: an integer
+
+This macro converts an ISO 8601-formatted date string into an SQL `DATE` type.
+ 
+**Usage**:
+
+
+**Sample output**:
+
+
```

Comment:
@dbeatty10 : is there a doc or something where i could grab info for the Usage and Sample output parts? unless i totally missed it, i wasn't able to find any from the issue nor the epic